cmd/sql.go

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Did you mean to commit this? It seems unrelated to CI

Reply:
Yes, this is re-adding the `pgroll sql` command that we had to remove in https://github.com/xataio/pgroll/pull/515.

We had to remove it because it caused [pgquery_go](https://github.com/pganalyze/pg_query_go) to be compiled as part of the project. `pgquery_go` uses `cgo` and so made compilation for different platforms more complicated than when `pgroll` was a pure-Go binary.

By removing this file the only dependency on `pgquery_go` was in tests. Now that we can compile with `CGO_ENABLED` on each of the platforms we need to target we can re-introduce the dependency.